modules/search/search.py

```diff
@@ -0,0 +1,38 @@
+import math
+import numpy as np
+from numpy import sin, cos
+
+
+def altSearch(tent_GPS, plane_GPS, angle):
+    """Search program to determine the minimum rotation angle between heading and tent
+
+    Parameters
+    ----------
+    tent_GPS : obj
+      dictionary containing lat and long coordinates of the tent
+    plane_GPS : obj
+      dictionary containing lat and long coordinate of the plane
+    angle : in
+      bearings notation in radians
+
+    Returns
+    -------
+    int
+      returns the minimum rotation angle
+
+    """
+    angle = math.radians(angle)
+    # vector of the plane's current heading
+    v_heading = np.array([cos(angle), sin(angle)])
+    # vector from plane to tent
+    v_plane_tent = np.array(
```

Comment:
We're standardizing the codebase on camelCase instead of snake_case, so please change variables like v_heading to vHeading.